share/scripts/inverse/functions_common.sh

```diff
@@ -642,22 +642,22 @@ add_to_csgshare() { #added an directory to the csg internal search directories
   for dirlist in "$@"; do
     old_IFS="$IFS"
     IFS=":"
```

Comment:
`$var` and `${var}` have identical meaning in bash! So, error message or it didn't happen!